keras_nlp/samplers/beam_sampler_test.py

```diff
@@ -38,7 +38,6 @@ def setUp(self):
                     output_dim=self.feature_size,
                 ),
                 keras.layers.Dense(self.vocab_size),
-                keras.layers.Softmax(),
```

Comment:
Why change this?

Reply:
It's because we by default set `from_logits=True` (was False earlier for mysterious reason), so I am aligning the unit test with it.